fluentui_listitem/src/main/java/com/microsoft/fluentui/tokenized/tabItem/TabItem.kt

```diff
@@ -117,7 +121,11 @@ fun TabItem(
     val iconContent: @Composable () -> Unit = {
         Icon(
             imageVector = icon,
-            modifier = Modifier.size(if (textAlignment == TabTextAlignment.NO_TEXT) 28.dp else 24.dp)
+            modifier = Modifier
+                        .semantics {
+                            invisibleToUser()
```

Comment:
why are we making this invisibleToUser?
And also this API is deprecated, refer this https://developer.android.com/reference/kotlin/androidx/compose/ui/semantics/package-summary#(androidx.compose.ui.semantics.SemanticsPropertyReceiver).invisibleToUser()

Reply:
Thanks for the suggestion, will update the API. 
This does not hide it from the user view. Everything will be visible as is, the UI will be same. It instead ignores it in the accessibility talkback announcement which allows us to override with our own custom talkback announcements only.

Reply:
This is deprecated very recently as part of the latest compose.ui 1.8.0-alpha06 build.

https://developer.android.com/jetpack/androidx/releases/compose-ui#1.8.0-alpha06

The latest stable release is 1.7.5 which does not support this. So we cannot implement the newer method at the moment. Will change the implementation when we upgrade to stable 1.8.0 or higher.